doc/INSTALL.md

```diff
@@ -24,7 +24,7 @@ For actually doing development and running the tests, you will also need:
 * pip3: to install python-bitcoinlib
 * valgrind: for extra debugging checks
 
-You will also need a version of bitcoind with segregated witness and `estimatesmartfee` economical node, such as the 0.16 or above.
+You will also need a version of bitcoind with segregated witness and `estimatesmartfee` economical mode, such as the 0.16 or above.
```

Comment:
"economical mode" still does not quite make sense to me.... maybe just say "and `estimatesmartfee` implementation"?

Reply:
The economical mode was added later. Maybe :
> and `estimatesmartfee` with economical mode
?

Reply:
"`estimatesmartfee` with "ECONOMICAL" fee estimate mode support"? (that "ECONOMICAL" in call caps and quotes as it is how it is called in `bitcoin-cli help estimatesmartfee` output). Maybe also could add that it's Bitcoin Core v0.15.0 or newer.

Reply:
It's specified 0.16 just after and that's 0.16 IIRC ? Like 0.15 `estimatesmartfee` and 0.16 `ECO` mode ?

Reply:
Ah the version was wrong too ! Thanks

Reply:
Yes, it was introduced in 0.15. https://bitcoin.org/en/release/v0.15.0#changes-to-fee-estimate-rpcs